crates/sui-core/src/db_checkpoint_handler.rs

```diff
@@ -218,6 +218,9 @@ impl DBCheckpointHandler {
         dirs.sort_by_key(|(epoch_num, _path)| *epoch_num);
         for (epoch, db_path) in dirs {
             if missing_epochs.contains(epoch) || *epoch >= last_missing_epoch {
+                // This writes a single "MANIFEST" file which contains a list of all files that make up a db snapshot
+                write_snapshot_manifest(db_path, self.input_object_store.clone()).await?;
```

Comment:
nit: if we just pass the `db.path.child("epoch_{}", epoch)` here, we can potentially avoid the trim operation in the function